apps/user_ldap/lib/User/User.php

```diff
@@ -249,9 +249,9 @@ public function processAttributes($ldapEntry) {
 			$profileValues = array();
 			//User Profile Field - Phone number
 			$attr = strtolower($this->connection->ldapAttributePhone);
-			if (isset($ldapEntry[$attr])) {
+			if (!empty($attr)) { // attribute configured
 				$profileValues[\OCP\Accounts\IAccountManager::PROPERTY_PHONE]
-					= $ldapEntry[$attr][0];
+					= (isset($ldapEntry[$attr]) ? $ldapEntry[$attr][0] : "");
```

Comment:
```suggestion
					= $ldapEntry[$attr][0] ?? "";
```
This should be enough no?